backend/db/utils.py

```diff
@@ -727,7 +706,8 @@ def list_tables(con: Connection = None, schema: str = None, filter_temp_refresh_
 
 
 def get_ddl_statements(
-    schema: str = SCHEMA, modules: List[str] = None, table_suffix='', index_suffix='', return_type=['flat', 'nested'][1]
+    schema: str = SCHEMA, modules: List[str] = None, table_suffix='',return_type=['flat', 'nested'][1],
+    unique_index_names=True,
```

Comment:
## Bugfix: Indexes not getting created
This was only happening for the vocab/counts refreshes. I forgot to pass a unique index hash, so it was detecting that an index (from the temp table that was about to get deleted) was already there, and not creating.

As part of this fix, I did a refactor on `get_ddl_statements()`, and basically made it so that when this function is called, the unique index names automatically get created.